### ISSUE TITLE
New channel becomes "White Zombie" if created with reserved word.

### ISSUE THREAD

**Issue by digitalextremist** [mattermost/mattermost-server#7756]
You cannot use/interact with channels created with reserved words such as `authorize`

#### Steps to reproduce
Create a channel with a name which is a reserved word, such as `authorize`

#### Expected behavior
Should have no issues: the word `authorize` is a perfectly good name for a channel.

#### Observed behavior
You will get a white screen upon creation, rather than joining the channel.
Upon returning to the Mattermost instance, returning to the channel causes same white screen:

![screenshot from 2017-11-01 14-14-45](https://user-images.githubusercontent.com/21095/32298880-cceec2fa-bf10-11e7-9f52-fb7b6ec12029.png)

Only way to remove channel becomes:

```
/opt/mattermost/bin$ sudo ./platform channel delete team:channel
```

**Comment by lindy65** [comment]
Hi @digitalextremist 

Thanks for your feedback,

Could you share which version of Mattermost you're using?

Reserved words for creating new teams [were removed](https://docs.mattermost.com/administration/config-settings.html?highlight=reserved%20word#restrict-team-names) in November 16, 2016 release and I'm not aware of reserved words for channel names.  As far as I'm aware, an error is displayed if you try to create a channel with an already existing channel name.

Perhaps I am misunderstanding your issue though?

**Comment by digitalextremist** [comment]
@lindy65 my apologies! Here is the `About Mattermost` screen:

![screenshot from 2017-11-02 03-28-00](https://user-images.githubusercontent.com/21095/32321293-e6105ec0-bf7d-11e7-9205-715871ab68ba.png)

This is about `channel` not `team` names.

Create a channel named `authorize` and see what happens.

**Comment by lindy65** [comment]
Hi @digitalextremist,

Thanks for your feedback and version,

I realise it's about channel names and did try to reproduce your issue earlier on our nightly build server but it didn't reproduce!  Now, however, it IS reproducing so I have created a [ticket](https://mattermost.atlassian.net/browse/PLT-8027) to track.

Sorry about the confusion, it just didn't reproduce for me earlier!

**Comment by digitalextremist** [comment]
No problem at all. I'm glad it's relevant!

**Comment by lindy65** [comment]
Thanks @digitalextremist 

I'll close this issue off here for now and we can track via the ticket linked above :)

**Comment by digitalextremist** [comment]
Much appreciated @lindy65. Will continue following at: 

* https://mattermost.atlassian.net/browse/PLT-8027

**Comment by lindy65** [comment]
FYI @digitalextremist - this is fixed now and should ship with v4.5 on Dec 16th.
